judoscale/celery/tasks_monitor.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from datetime import datetime as dt
+#from celery import Celery
+from celery import current_app
+
+from judoscale.core.metric import Metric
+from judoscale.core.reporter import reporter
+from judoscale.core.metrics_store import metrics_store
+
+logger = logging.getLogger(__name__)
+
+
+class EventLogger:
+    def log_task_status_change(self, task, event):
+        print('[{}] {} {} (STATE={}, UUID={})'.format(
+            self._to_datetime(task.timestamp),
+            event['type'].upper(),
+            task.name,
+            task.state.upper(),
+            task.uuid
+        ))
+    def _to_datetime(self, timestamp):
+        return dt.fromtimestamp(timestamp) if timestamp is not None else None
+
+
+class CeleryEventsHandler:
+    def __init__(self, celery_app, verbose_logging=False):
+        self._app = celery_app
+        self._state = celery_app.events.State()
+        self._logger = EventLogger()
+        self._verbose_logging = verbose_logging
+        self.task_started = {}
+        self.task_created = {}
```

Comment:
It looks like we're continually adding entries to these objects and never clearing them out. Is that right?